plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java

```diff
@@ -345,6 +352,8 @@ public Collection<Object> createComponents(
             nodeHelper
         );
 
+        mlModelAutoReLoader.autoReLoadModel();
```

Comment:
Do this in the MLModelAutoReloader constructor.

Reply:
done